exporter/prometheusexporter/end_to_end_test.go

```diff
@@ -41,7 +41,7 @@ func TestEndToEndSummarySupport(t *testing.T) {
 	}
 
 	// 1. Create the Prometheus scrape endpoint.
-	waitForScrape := make(chan bool, 1)
+	waitForScrape := make(chan bool, 10)
```

Comment:
This change doesn't belong here, does it?

Reply:
@Aneurysm9 oh no! sorry. I left it there by mistake. Didn't notice also. reverted it back to its original.